packages/atoms/connect/ConnectButton.tsx

```diff
@@ -8,14 +8,13 @@ import type { AtomsGlobalConfigProps } from "../types";
 interface ConnectButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   buttonText?: string;
   icon?: JSX.Element;
-  handleClick: () => void;
   onSuccess?: () => void;
   onError: () => void;
 }
 
 export function ConnectButton({
   buttonText,
-  handleClick,
+  onClick,
```

Comment:
@supalarry this is the correct way to do it right?

Reply:
Yeah i think this makes sense. Can you manually test it by using the component, passing onClick to it and seeing if it is fired when the button is clicked?

Reply:
Should work IMHO but will test and let you know!

Reply:
@supalarry I added a loader for the button can you review it again and make sure it's alright?

Reply:
Looks good!